src/randoop/sequence/MutableStatement.java

```diff
@@ -13,7 +13,7 @@
  */
```

Comment:
On line above, reference `MutableSequence`.  The MutableSequence class should indicate when one would choose to use each of the two types of sequence.  Also, its comment
<pre> * The "M" stands for "Mutable".</pre>
should be removed.